Kernel/FileSystem/RAMFS/FileSystem.h

```diff
@@ -13,24 +13,24 @@
 
 namespace Kernel {
 
-class TmpFS final : public FileSystem {
-    friend class TmpFSInode;
+class RAMFS : public FileSystem {
+    friend class RAMFSInode;
 
 public:
-    virtual ~TmpFS() override;
+    virtual ~RAMFS() override;
     static ErrorOr<NonnullLockRefPtr<FileSystem>> try_create();
     virtual ErrorOr<void> initialize() override;
 
-    virtual StringView class_name() const override { return "TmpFS"sv; }
+    virtual StringView class_name() const override { return "RAMFS"sv; }
 
     virtual bool supports_watchers() const override { return true; }
 
     virtual Inode& root_inode() override;
 
-private:
-    TmpFS();
+protected:
```

Comment:
This should still be `private:` I think?

Reply:
Yes! and also final. Fixed both :)